custom.webpack.config.js

```diff
@@ -3,6 +3,9 @@ var generated = require('./scalajs.webpack.config');
 
 var local = {
     module: {
+        devServer: {
```

Comment:
seems you need to move this one level up

Reply:
Possibly, I'll complete the PR once I get some free time.